examples/v0/reporting/get_hotel_ads_performance.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example gets Hotel-ads performance statistics for the 50 Hotel ad
+groups with the most impressions over the last 7 days.
+"""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+
+import google.ads.google_ads.client
+
+
+_DEFAULT_PAGE_SIZE = 50
+
+
+def main(client, customer_id, page_size):
+    ga_service = client.get_service('GoogleAdsService')
+
+    query = ('SELECT campaign.id, campaign.advertising_channel_type, '
+             'ad_group.id, ad_group.status, metrics.impressions, '
```

Comment:
In our other Python examples, we don't intent the subclauses of the SELECT and WHERE clauses, however we should probably start doing that as it is much easier to read (as in the PHP Golden)

Reply:
My experience so far with Python style is that it's more idiomatic to condense lists of items like this rather than put each one on a new line, though I also find it more difficult to read. However that normally applies more to parameter lists and function calls so it may be fine for a string value such as this. Ultimately I think we should just aim for consistency with the other examples and perhaps adjust them all in a separate change.